pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[project]
+name = "smpclient"
+version = "1.3.2"
```

Comment:
I'd prefer "0" if it is valid.  If not, then "0.0.0", or "0.1.0-dev", or anything to indicate that it is not a real version number.  This library uses the git tag as the SSOT for versioning so I'd rather give users that are installing unreleased versions no version rather than the wrong version.  Released version will always be available from PyPI.